packages/bits/spec/components/combobox-v2/combobox-v2.e2e.ts

```diff
@@ -117,6 +117,21 @@ describe("USERCONTROL Combobox >", () => {
 
                 await expect(await comboboxError.getInputValue()).toEqual("Item 0");
             });
+            
+            it("should focus on the first item in dropdown, when removing item on backspace", async () => {
+                await comboboxError.click();
+                const option = await comboboxError.getOption(5);
+                await option.click();
+                
+                await expect(await comboboxError.getInputValue()).toEqual("Item 5");
+                await expect(await (await comboboxError.getOption(5)).isActive()).toBe(true);
+
+                browser.actions().doubleClick(comboboxError.input);
```

Comment:
might need to resolve the returned promise explicitly here with await

Reply:
surprisingly, doubleclick does not return promise for some reason :) ignore the comment above